OrchardCore.sln

```diff
@@ -216,6 +216,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{184139CF
 	ProjectSection(SolutionItems) = preProject
 		src\OrchardCore.Build\Dependencies.AspNetCore.props = src\OrchardCore.Build\Dependencies.AspNetCore.props
 		src\OrchardCore.Build\Dependencies.props = src\OrchardCore.Build\Dependencies.props
+		src\OrchardCore\Directory.Build.targets = src\OrchardCore\Directory.Build.targets
```

Comment:
Is this one necessary?

Reply:
These are just to show them under the visual studio solution explorer view, so that we can find them easily for editing. @sebastienros already added the others, here I added a missing one.

Reply:
There are probably others missing. Could we get them all added to this PR then?

Reply:
Just tried under VStudio

That's okay for those that were already under a project folder because there were already listable.

For those that are not under a project folder, there are now all listable under the build solution folder